src/data-types/schemas.ts

```diff
@@ -0,0 +1,102 @@
+export const TxMAIngress = {
+  definitions: {},
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: 'https://example.com/object1689242392.json',
+  title: 'Encrypt Event Validator schema',
+  description: 'This schema validates the event that triggers Encrypt Data',
+  type: 'object',
+  required: ['event'],
+  properties: {
+    event: {
+      $id: '#root/event',
+      title: 'Event',
+      type: 'object',
+      required: ['timestamp', 'event_name', 'user'],
+      properties: {
+        timestamp: {
+          $id: '#root/event/timestamp',
+          title: 'Timestamp',
+          type: 'number',
+          examples: [1234],
```

Comment:
small question, we removed the examples field from the schema in vc storage, just to help improve compile time, should we do the same here as a slight optimisation?